pkg/inspector/image-inspector.go

```diff
@@ -190,9 +195,22 @@ func (i *defaultImageInspector) pullImage(client *docker.Client) error {
 	// Try all the possible auth's from the config file
 	var authErr error
 	for _, auth := range imagePullAuths.Configs {
+		ch := make(chan bool)
```

Comment:
since we're not giving any info about the specific auth being attempted does this need to live inside the for loop and create a new channel and routine for every iteration?